app/components/back_link_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item active">
+      <%= link_to @link do %>
+        <i class="breadcrumb-icon fa fa-angle-left mr-2"></i><%= @title %>
```

Comment:
I'm curious to the benefits of doing it this way. Is it more a style preference?

Reply:
Yes and also just being very specific about what’s available to avoid any unexpected visitors. It’s the same idea as passing variables down to your partials.